File: src/sdk/php/3/core-classes/memory-storage/pfadd/snippets/pfadd-1.php

<?php

use \Kuzzle\Kuzzle;


$kuzzle = new Kuzzle('localhost');

try {
  $status = $kuzzle->memoryStorage()->pfadd('key', ['foo', 'bar', 'baz']);
}
catch (ErrorException $e) {

}

File: src/sdk/java/2/core-classes/memory-storage/pfadd/snippets/pfadd-1.java

String[] elements = new String[]{"foo", "bar", "baz"};

kuzzle.memoryStorage.pfadd("key", elements, new ResponseListener<Boolean>() {
  @Override
  public void onSuccess(Boolean status) {
    // callback called once the action has completed
  }

  @Override
  public void onError(JSONObject error) {
  }
});
